MediaRecorder.html

```diff
@@ -469,6 +498,22 @@
         </dl>
       </section>
     </div>
+    <section>
+      <h2>Data handling</h2>
+        <p><dfn data-dfn-type="dfn" data-lt="fires a dataavailable event">
+        Firing a dataavailable event </dfn> with a <a
+        href="https://www.w3.org/TR/FileAPI/#blob"><code>Blob</code></a>
+        <var>blob</var> means <a
+        href="https://dom.spec.whatwg.org/#concept-event-fire">firing</a>
+        an event named <code>dataavailable</code> at the current
+        <code>MediaRecorder</code> object using a <a>BlobEvent</a> with its
+        <code>data</code> initialized to <var>blob</var>.
+        Usually <var>blob</var> will be the data gathered by the UA after the
+        last transition to <a href="#dom-recordingstate-recording">recording
```

Comment:
I don't think this last sentence holds since state doesn't change when *timeslice* is up, or `requestData` is called. Maybe leave out that sentence?